etc/config/.env.example

```diff
@@ -32,7 +32,6 @@ BROWSER=chrome
 
 #*** Uncomment and set vault base url and access token if you want to use vault to manage _CREDS secrets ***#
 #CREDENTIAL_VAULT_BASE_URL=
-#CREDENTIAL_VAULT_TOKEN=
```

Comment:
I don't think we should add VAULT_CONFIG_PATH in MFTF. For example, if user set this env in MFTF, we are not supposed to load it, nor it will work with vault.